modules/presentation.py

```diff
@@ -0,0 +1,232 @@
+            lambda tweet: sentiment != tweet["sentiment_analysis"]["verdict"], tweets
+        )
+    )
+
+
+# FILTERING / SORTING FUNCTIONS DONE
+
+# 2. GET SENTIMENT
+def get_sentiment(tweets):
+    """
+    Filter tweets using other methods, before you use this one. 
+    Makes a list of tweets into a Pandas DataFrame. 
+    It has date on index and Sentiment on Columns and the value is the amount of tweets. 
+    """
+    # Make a dict, where key is date and value is a list of all sentiments for that date
+    tweets_dict = defaultdict(list)
+    for tweet in tweets:
+        tweets_dict[tweet["date"]].append(tweet["sentiment_analysis"]["verdict"])
+
+    # Takes .value_counts() https://www.geeksforgeeks.org/python-pandas-index-value_counts/
+    for date in tweets_dict.keys():
+        tweets_dict[date] = pd.Series(tweets_dict[date]).value_counts()
+
+    # Makes a transposed dataframe sorted by index(date in this case)
+    return pd.DataFrame(tweets_dict).T.sort_index()
+
+
+# 3. PLOT
+# PLOTTING START
+def pie_chart(df, title, save=None):
+    """
+    Make a Pie Chart about Sentiments for given dataframe
+
+    Parameters: \n
+        df = DataFrame made by get_sentiment
+        title = String
+        save = If set, save with this file_name
+    """
+    sentiment = {}
+    for column in sentiments:
+        if column in df.columns:
+            sentiment[column] = df[column].sum()
+
+    plt.axes(pd.Series(sentiment).plot(kind="pie", autopct="%1.0f%%", title=title))
+    plt.ylabel("")
+
+    if save:
+        save_plot(plt, save)
+    else:
+        plt.show()
+
+
+def bar_plot(df, title, save=None):
+    """
+    Make a Bar Plot about Sentiments for given dataframe
+
+    Parameters: \n
+        df = DataFrame made by get_sentiment
+        title = String
+        save = If set, save with this file_name
+    """
+    df.plot(kind="bar", rot=0, title=title)
+
+    if save:
+        save_plot(plt, save)
+    else:
+        plt.show()
+
+
+def line_plot(df, title, save=None):
+    """
+    Make a Line Chart about Sentiments for given dataframe
+
+    Parameters: \n
+        df = DataFrame made by get_sentiment
+        title = String
+        save = If set, save with this file_name
+    """
+    df.plot(kind="line", title=title)
+
+    if save:
+        save_plot(plt, save)
+    else:
+        plt.show()
+
+
+# PLOTTING END
+
+# Save helper function 
+def save_plot(fig, name):
+    """
+    Save Plot to file
+
+    Parameters:\n
+        fig: Figure. pyplot fig. 
+        name: string. Name of file. No extension. 
+
+    Returns:\n
+        Nothing. 
+    """
+    if isinstance(name, str):
+        from pathlib import Path
+
+        # https://stackoverflow.com/a/273227/11255140
+        # if plots folder doesn't exist, create it.
+        Path("./plots").mkdir(parents=True, exist_ok=True)
+        # bbox_inces="tight" ensures less white space in the image.
+        fig.savefig("./plots/" + name + ".png", bbox_inces="tight")
```

Comment:
shouldn't this be bbox_inc**h**es? haven't run it locally to test

Reply:
omg so thats why I felt like it didnt do anything

Reply:
Fixed in #996ddd4